keps/sig-auth/4940-psa-block-host-field-in-probes/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: KEP Template
+kep-number: 4940
+authors:
+  - "@tssurya"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-network
+status: provisional
+creation-date: 2024-10-31
+reviewers:
+  - deads2k
+approvers:
+  - deads2k
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.33"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.33"
+  beta: "v1.35"
+  stable: "v1.37"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: N/A We decided to go with PSA versioning
```

Comment:
don't put freeform text in `kep.yaml`; some scripts try to interpret it. Just leave this empty.